bin/app.ts

```diff
@@ -38,11 +38,23 @@ class ServiceDeployIAM extends cdk.Stack {
           const cloudFormationStackResource = ``
           const eventBridgeResources = [`arn:aws:events:${region}:${accountId}:rule/${serviceName}*`]
           const s3DeploymentResources = [`arn:aws:s3:::${serviceName}*serverlessdeployment*`]
+          const s3DeploymentObjectResources = [`arn:aws:s3:::${serviceName}*serverlessdeployment*/*`]
           const ssmDeploymentResources = [`arn:aws:ssm:${region}:${accountId}:parameter/${serviceName}*`]
           const serviceRole = new Role(this, `ServiceRole-v${version}`, {
                assumedBy: new ServicePrincipal('cloudformation.amazonaws.com')
           });
 
+          // S3 Deployment Bucket managment
+          serviceRole.addToPolicy(
+               new PolicyStatement({
+                    effect: Effect.ALLOW,
+                    resources: s3DeploymentResources,
+                    actions: [
+                         "s3:*",
```

Comment:
Should we restrict this to just bucket creation, deletion and whatever we know it needs? * gives it a lot of extra stuff that I'm not sure we need?

Reply:
I did, think about this but assumed the deployment bucket was essentially "owned" by the stack.
I will make it more restrictive.